src/action.c

```diff
@@ -824,6 +832,53 @@ get_target_output(struct output *output, struct server *server,
 	return target;
 }
 
+static void
+warp_cursor(struct view *view, struct output *output, const char *to, const char *x, const char *y)
+{
+	struct wlr_box target_area = {};
+	int goto_x;
+	int goto_y;
+
+	if (!strcasecmp(to, "output") && output) {
+		target_area = output->usable_area;
+	} else if (!strcasecmp(to, "view") && view) {
```

Comment:
"View" is just an internal term and we should not expose it to users. "Window" should be suitable here.